### PR TITLE
Support for older versions of Sidekiq

### PR DESCRIPTION
**Pull request by potomak**

### CODE REVIEW

File: sidekiq-debounce.gemspec [hunk 1]
```diff
@@ -25,10 +25,11 @@ DESC
...
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sidekiq', '>= 3.1.0'
+  spec.add_dependency 'sidekiq', '>= 2.17'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'mock_redis'
   spec.add_development_dependency 'mocha'
   spec.add_development_dependency 'codeclimate-test-reporter'
+  spec.add_development_dependency 'minitest'
```
Comment (NuckChorris) [inline sidekiq-debounce.gemspec:34]: The commit message says there was a problem requiring `minitest/spec`, but that should ship with modern rubies.  What's your Ruby flavor+version?  We might wanna add it to the build matrix

**Review by NuckChorris — COMMENTED** [review]
Overall looks good, I don't see anything wrong with supporting Sidekiq 2.17 if it's this easy

File: sidekiq-debounce.gemspec (hunk 1 above)
Comment (potomak) [reply sidekiq-debounce.gemspec:34]: I used MRI 2.2.2.

Comment (NuckChorris) [reply sidekiq-debounce.gemspec:34]: Strange.  Well, I don't see any problems with including minitest itself, so :+1:

**Comment by potomak** [comment]
@NuckChorris I don't think tests failure are connected to my update. Do you know how to fix Travis' configuration?

**Comment by NuckChorris** [comment]
It looks like Travis no longer adds aliases for MAJOR.MINOR, so we have to specify the TEENY

So I guess the fix for this is to make `2.2` into `2.2.5` and `2.3` into `2.3.1` like Rails has :/

**Comment by potomak** [comment]
@NuckChorris thanks for the suggestion, I committed a new version of Travis configuration and builds succeeded.

**Review by NuckChorris — APPROVED** [review]
Looks great

**Comment by NuckChorris** [comment]
Released as sidekiq-debounce 1.1.0 :thumbsup:

**Pull request merged**